### PR TITLE
update report summary test location

### LINKED ISSUE #478: Change tests summary report location
### Description the feature
Current the tests summary is located at the end of report, generally a summary should be located at the start of a document to provide a clearly overview.

### Is your feature request related to a problem?
No

### Do you have any suggestions on how to add this feature in scanapi ?
An easy way to fix this is just change the order of rendered elements with CSS.

### How it is
image

### How could be
image

### PR DESCRIPTION
**Pull request by GitVitor**
## Description
update report summary test location
update report classes following BEMCSS pattern
close unclosed html tags

## Motivation behind this PR?
[478](https://github.com/scanapi/scanapi/issues/478)

## What type of change is this?
Feature

## Checklist
 

- [x]  I have added a changelog entry / my PR does not need a new changelog entry. [Instructions](https://github.com/scanapi/scanapi/wiki/Changelog).
- [ ] I have added/updated unit tests. [Instructions](https://github.com/scanapi/scanapi/wiki/Writing-Tests).
- [x] New and existing unit tests pass locally with my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally#tests)
- [ ] I have self-documented code my changes by adding docstring(s) and comment(s). [Instructions](https://github.com/scanapi/scanapi/wiki/First-Pull-Request#7-make-your-changes)
- [x] Current PR does not significantly decrease the code coverage and docstring coverage.
- [x] My code follows the style guidelines of this project.
- [x] I have run ScanAPI locally and manually tested my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally).

## Issue

Closes #478

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Fixed
```
Comment (camilamaia) [inline CHANGELOG.md:9]:
```suggestion
### Changed
- Summary tests location to the top of the report [#479](https://github.com/scanapi/scanapi/pull/479)

### Fixed
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Header table gets broken. [#432](https://github.com/scanapi/scanapi/pull/432)
+- Change summary tests location. [#479](https://github.com/scanapi/scanapi/pull/479)
```
Comment (camilamaia) [inline CHANGELOG.md:11]:
```suggestion
```

**Review by camilamaia — CHANGES_REQUESTED** [review]
Thank you very much for the contribution!!

I left just a minor suggestion related to the CHANGELOG. Otherwise everything is great ⭐️ This position is much better 🎨

**Comment by GitVitor** [comment]
> Thank you very much for the contribution!!
> 
> I left just a minor suggestion related to the CHANGELOG. Otherwise everything is great ⭐️ This position is much better 🎨

Suggestion merged

**Review by camilamaia — APPROVED** [review]

**Comment by camilamaia** [comment]
@GitVitor Congrats for your first PR merged! Thank you very much!

I am going to send you an invite to join the ScanAPI org on GitHub 🚀 We invite everyone that has contributed with a merged PR in any of our repos. Here you can check our [Contributing Guidelines](https://github.com/scanapi/contributors/blob/main/CONTRIBUTING.md) so you can understand better how it works.

And once again, thank you! 🙇‍♀️

**Pull request merged**